hack/tools/configure_vault_kind_multi.sh

```diff
@@ -2,6 +2,13 @@
 # Copyright 2020 IBM Corp.
 # SPDX-License-Identifier: Apache-2.0
 
+# This script configures Vault which is deployed on the control-plane namespace
```

Comment:
I would write the following to not have people confused that might look for a namespace called `control-plane`:

`This script configures Vault which is deployed on the control-plane namespace (usually m4d-system)`

Reply:
Will change that Thanks